pkg/pkgupdates/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<!--
+This file is part of Cockpit.
+
+Copyright (C) 2017 Red Hat, Inc.
+
+Cockpit is free software; you can redistribute it and/or modify it
+under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+Cockpit is distributed in the hope that it will be useful, but
+WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+-->
+<html>
+
+<head>
+  <title translate>Software Updates</title>
```

Comment:
We don't set `window.title` anywhere in Cockpit. It's not really necessary either, as even without it the window title on http://localhost:9090/updates is fine, it's apparently built from the manifest's tools→updates→label plus the hostname. Do you know if setting it explicitly is useful for anything? Perhaps embedding as iframe in third-party pages or so? As every other pkg/* has a `<title>` I'm a bit hesitant to drop it.